msteams-platform/toolkit/debug-background-process.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Debug background processes
+author: zyxiaoyuer
+description: Function of Visual studio code and Teams Toolkit during local debug
+ms.author: surbhigupta
+ms.localizationpriority: high
+ms.topic: overview
+ms.date: 03/03/2022
+---
+
+# Debug background process
+
+The local debug workflow involves the `.vscode/launch.json` and `.vscode/tasks.json` files to configure the debugger in Visual Studio Code, then the Visual Studio Code launches the debuggers, and  Microsoft Edge or Chrome debugger launches a new browser instance as follows:
+
+1. The `launch.json` file configures the debugger in Visual Studio Code. Visual Studio Code runs the compound configurations `preLaunchTask`, defined as `Pre Debug Check`, and `Start All` in `.vscode/tasks.json` file.
+
+2. Visual Studio Code launches the debuggers specified in the compound configurations, such as **Attach to Bot**, **Attach to Backend**, **Attach to Frontend**, and **Launch Bot**.
+
+3. Microsoft Edge or Chrome debugger launches a new browser instance and opens a web page to load Teams client.
+
+## Prerequisites
+
+
+Visual Studio Code checks all prerequisites in the process of debug actions:
+
+* Node.js is installed, applicable for project types, such as tab without Azure functions, tab with Azure functions ( versions 10, 12, **14 (recommended)**), bot and messaging extensions,  versions 10, 12, **14 (recommended)**, 16.
```

Comment:
It would be clearer if using a table like:
![image](https://user-images.githubusercontent.com/37978464/158822259-26030815-8485-4d2b-b939-71ad3922c63d.png)

Reply:
Thanks. @v-bdeepa, please update a table.